system_metrics_collector/src/system_metrics_collector/default_config.hpp

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef SYSTEM_METRICS_COLLECTOR__DEFAULT_CONFIG_HPP_
+#define SYSTEM_METRICS_COLLECTOR__DEFAULT_CONFIG_HPP_
+
+#include <chrono>
+
+namespace system_metrics_collector
+{
+
+constexpr const char kStatisticsTopicName[] = "system_metrics";
+constexpr const std::chrono::seconds kDefaultCollectPeriod{1};
+constexpr const std::chrono::minutes kDefaultPublishPeriod{1};
```

Comment:
I found a StackOverflow on this: https://stackoverflow.com/questions/50488831/use-of-constexpr-in-header-file
It looks like an alternative would be to wrap `kStatisticsTopicName` in a function.
`kDefaultCollectPeriod` and `kDefaultPublishPeriod` are probably fine since an address is not required when either are accessed.

Reply:
More on this: https://groups.google.com/a/chromium.org/forum/#!topic/chromium-dev/zu1NMS0RyrM
`constexpr const char kStringLiteral[] = ...` probably should require a .cpp file when defined in a header since it does have external linkage in C++14.

Reply:
This is no longer part of this PR.